### ISSUE TITLE
missing of query stream model

### ISSUE THREAD

**Issue by huangjy-pku** [cliport/cliport#18]
Hello, I met another confusion that, in one-stream language-goal transport module, both `logits` and `kernel` are derived by passing through `key_stream_one`, with the other stream model (`query_stream_one`) unused. I don't know if it is a typo, but referring to the two-stream transport module, where the `kernel` is derived by passing through `query_stream`, I suspect it needs fixing here.

The code is here: https://github.com/cliport/cliport/blob/152610b6913a84b5d67a356e6b80c591d80d7d7a/cliport/models/streams/one_stream_transport_lang_fusion.py#L23

**Comment by MohitShridhar** [comment]
@huangjy-pku, thanks for catching this. This is a typo.
Fixed in https://github.com/cliport/cliport/commit/47bf8298f6b664365ab042eb7ac0bcaf5df60254.
